marlowe-playground-client/test/Marlowe/LintTests.purs

```diff
@@ -312,7 +312,7 @@ unreachableCaseNotify =
 unreachableCaseEmptyChoiceList :: Test
 unreachableCaseEmptyChoiceList =
   testWarningSimple "When [Case (Choice (ChoiceId \"choice\" (Role \"alice\")) []) Close] 10 Close"
-    "This case will never be used, because there are no options to choose"
+    $ show UnreachableCaseEmptyChoice
```

Comment:
Is this `UnreachableCaseEmptyChoice` show instance overloaded or is it going to just show "UnreachableCaseEmptyChoice"?

Reply:
it shows the full message, I used show in the test because I changed it and thought there is not much point having a test to test a show instance.

Reply:
Ah, ok, sorry